packages/rs-drive-abci/src/execution/platform_events/core_chain_lock/verify_chain_lock_locally/v0/mod.rs

```diff
@@ -150,14 +152,19 @@ where
 
                 let mut engine = sha256d::Hash::engine();
 
-                engine.input(&[self.config.chain_lock_quorum_type() as u8]);
-                engine.input(quorum_hash.as_slice());
+                let mut reversed_quorum_hash = quorum_hash.to_byte_array().to_vec();
+                reversed_quorum_hash.reverse();
```

Comment:
why is this now reversed?

Reply:
We have done it before as well but inside choose_quorum function. Now it returns QuorumHash, Quorum for consitency (as all other functions there).